### PR TITLE
Fixes for properly setting parameters

### PR DESCRIPTION
**Pull request by matheusdiogenesandrade**
This is the commit of the issue #9.

### CODE REVIEW

File: Manifest.toml
Comment (dourouc05) [inline Manifest.toml:1]: The `Manifest.toml` should never be committed, at least for a package. It's only useful to ensure that you can get exactly the same environment as you currently have, which is not exactly a feature for packages.

File: test/api_java.jl [hunk 1]
```diff
@@ -1,4 +1,86 @@
...
+    function set_parameters(model::JavaCPOModel)
+        # Set parameters (DefaultInferenceLevel)
+        # Be careful when setting the parameter value to "Default", 
+        # because it allows the IloCP to control the value of the 
+        # parameter, as it is for the parameter 
+        # `AllMinDistanceInferenceLevel` 
+        # (https://www.ibm.com/docs/api/v1/content/SSSA5P_20.1.0/ilog.odms.cpo.help/refjavacpoptimizer/html/ilog/cp/IloCP.IntParam.html#AllMinDistanceInferenceLevel) 
+        # for instance. And thus, the parameter value retrieval may not
+        # be the same as the value entered.
+        int_parameters = Dict{String, Union{Int32, String}}(
```
Comment (dourouc05) [inline test/api_java.jl]: Is there a reason why you have a dictionary for integer and strings, then another one for doubles? Couldn't you have one dictionary for all three use cases?

File: test/api_java.jl [hunk 2]
```diff
@@ -87,6 +172,9 @@
             i2 = cpo_java_intervalvar(model)
             @test typeof(cpo_java_intervalsequencevar(model, [i1, i2])) ==
                   IloIntervalSequenceVar
+
+            # Set parameters
+            set_parameters(model)
```
Comment (dourouc05) [inline test/api_java.jl]: Is it useful to test this here? (It's the only place where I wouldn't have called that function in this PR.)

**Review by dourouc05 — CHANGES_REQUESTED** [review]
Generally good as it stands!

File: Manifest.toml
Comment (matheusdiogenesandrade) [reply Manifest.toml:1]: Thanks. Change implemented.

File: test/api_java.jl (hunk 1 above)
Comment (matheusdiogenesandrade) [reply test/api_java.jl]: No particular reason. Change implemented [here](https://github.com/JuliaConstraints/CPLEXCP.jl/pull/10/commits/57e31025321f8fcc141b951c51cfd5c27c4676e4#diff-a96c425fa0553a595e9ce870fae44377a4d7077b474b08b3e6d7bd81162eb0e8R12).

File: test/api_java.jl (hunk 2 above)
Comment (matheusdiogenesandrade) [reply test/api_java.jl]: No it isn't. [Link](https://github.com/JuliaConstraints/CPLEXCP.jl/pull/10/commits/57e31025321f8fcc141b951c51cfd5c27c4676e4#diff-a96c425fa0553a595e9ce870fae44377a4d7077b474b08b3e6d7bd81162eb0e8L175).

**Comment by matheusdiogenesandrade** [comment]
Let me know case there is any change from my side. Regards.

**Review by dourouc05 — APPROVED** [review]

**Pull request merged**